src/main/scala/nl/biopet/utils/ngs/bam/package.scala

```diff
@@ -194,4 +194,25 @@ package object bam {
     reader.close()
     dict
   }
+
+  /**
+    * This will retrieve the [[SAMSequenceDictionary]] from the bam file.
+    * When `referenceFasta is given he will validate this against the bam file.`
+    */
+  def validateReferenceInBam(
```

Comment:
Can this method be split up in 2 method so it also accepts `SamHeader` as input?

Reply:
On it

Reply:
👍